charts/victoria-logs-single/Chart.yaml

```diff
@@ -4,7 +4,27 @@ description: Victoria Logs Single version - high-performance, cost-effective and
 name: victoria-logs-single
 version: 0.3.0
 sources:
-- https://github.com/VictoriaMetrics/helm-charts
+  - https://github.com/VictoriaMetrics/helm-charts
+icon: https://avatars.githubusercontent.com/u/43720803?s=200&v=4
+kubeVersion: ">=1.23.0"
```

Comment:
Should we set kubeVersion with ">=1.23.0-0"? 
Since there could be some problem with helm's semverCompare, see https://github.com/helm/helm/issues/10375.

Reply:
You're right, fixed